docs/release-notes/highlights-6.3.0.asciidoc

```diff
@@ -1,9 +1,140 @@
 [[release-highlights-6.3.0]]
-== 6.3.0 release highlights
+== 6.3.0 Release Highlights
 ++++
 <titleabbrev>6.3.0</titleabbrev>
 ++++
 
 coming[6.3.0]
 
-See also <<breaking-changes>> and <<release-notes-6.3.0>>.
\ No newline at end of file
+Each release of {kib} brings new features and product improvements.
+Here are the highlights of the features that were added in 6.3 and
+how the user experience improved.
+
+Refer to the <<release-notes-6.3.0,6.3.0 Release Notes>> for a list bug
```

Comment:
nit: I'm not sure if the title is necessary here. I think if you just list `<<release-notes-6.3.0>>`, it'll automatically pull the title from the corresponding doc so they are guaranteed to be consistent.

This is true of pretty much all the `<<>>` links in this file.